### ISSUE TITLE
Inspec exec report for inherited profiles with different name

### ISSUE THREAD

**Issue by alexpop** [inspec/inspec#3135]
### 🎛 Description

🐛 bug report

The inspec report for a wrapper profile with the following `inspec.yml`:
```
name: cis-rhel7-level2-server-wrapper
...
depends:
  - name: cis-rhel7-level2-server-v2.1.1
    compliance: admin/cis-rhel7-level2-server
```

will have this content:
```
{
  "profiles": [
    {
      "name": "cis-rhel7-level2-server-wrapper",
      "depends": [
        {
          "name": "cis-rhel7-level2-server-v2.1.1",
          "compliance": "admin/cis-rhel7-level2-server"
        }]
      ...
    },
    {
      "name": "cis-rhel7-level2-server",
      "version": "2.1.1-18",
      ...
    }
  ]
}
```

without an easy way to link depends name `cis-rhel7-level2-server-v2.1.1` with inherited profile name `cis-rhel7-level2-server`
### 🌍 InSpec and Platform Version

```
inspec version 2.1.81
```

### 💁 Possible Solutions

Add a `profile_name` or similar field in depends with the actual name of the profile: `cis-rhel7-level2-server`

**Comment by clintoncwolfe** [comment]
@jquick - is this an example of the "inheritance json report" issue?

**Comment by jquick** [comment]
This will be fixed by the inheritance redo in 3.0 but is a smaller subset of an issue. For now we are going to add another field into the json report so A2 can match child profiles accordingly.

**Comment by jquick** [comment]
This actually may be a little difficult. The profile has no idea what its upstream name is or its location. So we can do some fuzzy matching but I don't see an easy 1:1 map here. May end up having to inject them during the loading.
